File: 예상_대진표.py
def solution(n,a,b):
    answer = 0
    large=max(a,b)
    small=min(a,b)
    while(True):
        answer+=1
        if large%2==0 and large==small+1:
            break
        large//=2
        small//=2

    return answer

File: golang/Level 2/예상_대진표.go
func solution(n int, a int, b int) int {
    answer := 0
    var large, small int
    
    if a > b {
        large = a
        small = b
    } else {
        large = b
        small = a
    }
    
    for {
        answer++
        if large % 2 == 0 && large == small + 1 {
            break
        }
        large = int((large + 1)/2)
        small = int((small + 1)/2)
    }

    return answer
}
